client/src/views/User.vue

```diff
@@ -100,19 +91,31 @@ export default Vue.extend({
   },
   data() {
     return {
-      activeTab: 0
+      activeTab: 0,
+      tabContentHeight: 0,
+      tabs: [
+        { id: 0, icon: "fa fa-award", component: "user-achievements" },
+        { id: 1, icon: "fa fa-users", component: "user-friends" },
+        { id: 2, icon: "fa fa-trophy", component: "user-leaderboard" },
+        { id: 3, icon: "fa fa-cog", component: "user-settings" }
+      ]
     };
   },
   computed: {
     ...mapState("user", ["user"])
   },
+  mounted() {
+    this.tabContentHeight =
+      this.$refs.userWrapper?.clientHeight - this.$refs.tabs?.clientHeight;
```

Comment:
Curious, what is the reason for using `$refs` with `ref` instead of `querySelector` with `.classname`?

Reply:
https://forum.vuejs.org/t/difference-between-refs-and-queryselector/35504/2